### PR TITLE
Test/again

### PR DESCRIPTION
**Pull request by kgds-dev**
This pull request is for testing purposes only

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <body>
     <script src="script.js"></script>
     <button>TESTER</button>
+    <span>andito ako sa testing only branch</span>
```
Comment (kgds-dev) [inline index.html]: This is wrong can you change this to this and resubmit a review

**Comment by LASER-East** [comment]
This is awesome!

File: index.html (hunk 1 above)
Comment (LASER-East) [inline index.html]: hindi kita makita dito.. ahaha

Comment (jsiroy) [reply index.html]: noted

**Review by LASER-East — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kgds-dev) [reply index.html]: got it

**Review by LASER-East — APPROVED** [review]
This is approved!

**Pull request merged**